libbeat/dashboards/kibana_loader.go

```diff
@@ -58,14 +60,13 @@ type KibanaLoader struct {
 
 // NewKibanaLoader creates a new loader to load Kibana files
 func NewKibanaLoader(ctx context.Context, cfg *config.C, dashboardsConfig *Config, hostname string, msgOutputter MessageOutputter, beatname string) (*KibanaLoader, error) {
-
 	if cfg == nil || !cfg.Enabled() {
-		return nil, fmt.Errorf("Kibana is not configured or enabled")
+		return nil, fmt.Errorf("kibana is not configured or enabled")
```

Comment:
```suggestion
		return nil, fmt.Errorf("Kibana is not configured or enabled")
```

I would silence the lint warning for this one since this is a user facing message containing a product name